internal/resourcepolicies/volume_resources.go

```diff
@@ -0,0 +1,201 @@
+package resourcepolicies
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+	corev1api "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+type volumePolicy struct {
```

Comment:
Make this type as exported

Reply:
As this struct is used internally, it may not need to exported